edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudConnection.cs

```diff
@@ -118,10 +118,11 @@ protected async Task<ICloudProxy> CreateNewCloudProxyAsync(ITokenProvider newTok
 
         async Task<IClient> ConnectToIoTHub(ITokenProvider newTokenProvider)
         {
-            Events.AttemptingConnectionWithTransport(this.transportSettingsList, this.Identity, this.modelId);
-            IClient client = this.modelId.Match(
-                m => this.clientProvider.Create(this.Identity, newTokenProvider, this.transportSettingsList, m),
-                () => this.clientProvider.Create(this.Identity, newTokenProvider, this.transportSettingsList));
+            this.modelId.ForEach(
```

Comment:
Move this logic to Events

Reply:
the idea is to avoid logging logic in the main code

Reply:
So leave it as it was, correct?